### PR TITLE
New dev env section

### LINKED ISSUE #1939: Development environment section in guides
## Context

https://prisma-company.slack.com/archives/C5Z9TH6N9/p1624520976365200?thread_ts=1624464292.357500&cid=C5Z9TH6N9

## Suggested action

A new section under Guides named "Development environment" that includes:

~~* Env vars (currently under 'More')~~
* Workspaces #1910 
~~* Editor setup (currently under 'More')~~

### PR DESCRIPTION
**Pull request by molebox**
Added a new Development environment section. This will house such things as working with env vars, editor setup and other dev env related things. 

Fixes: #1939 

Once merged, merge the redirects PR: https://github.com/prisma/homepage-v8/pull/311

### CODE REVIEW

**Review by janpio — CHANGES_REQUESTED** [review]
Environment variables are not a "development environment" concern, they also affect deployments.

**Comment by molebox** [comment]
> Environment variables are not a "development environment" concern, they also affect deployments.

It was decided a while ago, by none other than yourself and martina 😅 That while not 100% ideal, placing env vars info in a new section called development environment would be a step in the right direction: https://prisma-company.slack.com/archives/C5Z9TH6N9/p1624520976365200?thread_ts=1624464292.357500&cid=C5Z9TH6N9

**Review by thebiglabasky — APPROVED** [review]
Proposed a tiny fix, but LGTM otherwise.

File: content/300-guides/125-development-environment/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: 'Development environment'
+navTitle: Development environment
+metaTitle: 'Development environment'
+metaDescription: 'Using env vars, editors and workspaces in your development environment'
+staticLink: true
+---
+
+<TopBlock>
+
+This section describes using `env` vars, editors and workspaces in your development environment.
```
[the remark this answers is not in the document]
Comment (molebox) [reply content/300-guides/125-development-environment/index.mdx]: Ah yes! This was copilot. And there i was, so impressed that i missed the grammer! 😅

**Pull request merged**